DUI3-DX/Converters/Revit/Speckle.Converters.RevitShared/Raw/BoundarySegmentConversionToSpeckle.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Objects;
+using Speckle.Converters.Common.Objects;
+
+namespace Speckle.Converters.RevitShared.ToSpeckle;
+
+public class BoundarySegmentConversionToSpeckle : IRawConversion<IList<DB.BoundarySegment>, SOG.Polycurve>
+{
+  private readonly IRawConversion<DB.Curve, ICurve> _curveConverter;
+
+  public BoundarySegmentConversionToSpeckle(IRawConversion<DB.Curve, ICurve> curveConverter)
+  {
+    _curveConverter = curveConverter;
+  }
+
+  public SOG.Polycurve RawConvert(IList<DB.BoundarySegment> target)
+  {
+    if (!target.Any())
```

Comment:
very small thing, but the compiler will suggest that you prefer Count == 0 to .Any() for lists or arrays